src/com/google/foam/demos/olympics/Controller.js

```diff
@@ -24,7 +24,7 @@ foam.CLASS({
     'foam.dao.MDAO',
```

Comment:
Ignore file: Feature required universal HTTPRequest. Will merge in #248 after it's reviewed.